ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml

```diff
@@ -71,7 +64,7 @@ tests:
     TESTS_TO_FOCUS=$(grep -E -o '\[crit\:high\]' tests/*_test.go | sort | uniq | sed -E 's/tests\/([a-z_]+)\.go\:.*/\1/' | tr '\n' '|' | sed 's/|$//')
     FUNC_TEST_ARGS='--ginkgo.noColor --ginkgo.focus='"$TESTS_TO_FOCUS"' --ginkgo.regexScansFilePath=true --junit-output='"$ARTIFACT_DIR"'/junit.functest.xml' \
         bash -x ./hack/functests.sh
-
+  cron: 3 0 * * *
```

Comment:
I wonder if the time on our machines is synced with the time on GCP... if not, we need to adjust the times.
Also, I'm not sure that 45m are enough space between the triggers. Maybe give it 2 hours?

Reply:
Good point. Although I think all times on CRON taks should be UTC based, let's check this.